LayoutTests/imported/w3c/web-platform-tests/html/rendering/non-replaced-elements/form-controls/resets.html

```diff
@@ -59,6 +59,7 @@
 td, th { display: table-cell; }
 table {
   text-indent: initial;
+  box-sizing: border-box;
```

Comment:
This is already in above rule, in: `table { display: table; box-sizing: border-box; }`

Is it really needed?

Reply:
not needed. Indeed. I can remove it and push again.